tests/datasource/conftest.py

```diff
@@ -1,21 +1,101 @@
 import hashlib
 import math
 import os
+from itertools import product
 
 import pytest
+from moto import mock_glue
 
 from great_expectations.data_context.util import file_relative_path
 from great_expectations.datasource import (
     PandasDatasource,
     SparkDFDatasource,
     SqlAlchemyDatasource,
 )
+from great_expectations.execution_engine.sparkdf_execution_engine import (
+    SparkDFExecutionEngine,
+)
 from great_expectations.execution_engine.sqlalchemy_execution_engine import (
     SqlAlchemyExecutionEngine,
 )
 from great_expectations.self_check.util import get_sqlite_connection_url
 
 
+def create_partitions_for_table(
```

Comment:
❤️

Reply:
@lccasagrande A brief docstring explaining what these partitions are and why they are needed/useful would be really nice!  Thank you!

Reply:
Done